.gitignore

```diff
@@ -77,3 +77,5 @@
 /backup-routes.py
 /worke.py
 /myrobotlab*.jar
+/libbulletjme.so
```

Comment:
these shouldn't be added to the git ignore..

Reply:
they should be checked in ?  I doubt it ...